unity-client/Assets/Scripts/MainScripts/DCL/Components/Audio/DCLAudioStream.cs

```diff
@@ -117,5 +118,20 @@ private void StartStreaming()
             isPlaying = true;
             Interface.WebInterface.SendAudioStreamEvent(model.url, true, model.volume * settingsVolume);
         }
+
+        public void UpdateOutOfBoundariesState(bool enable)
+        {
+            if (!isPlaying)
+                return;
+
+            if (enable)
+            {
+                StartStreaming();
+            }
+            else
+            {
+                Interface.WebInterface.SendAudioStreamEvent(model.url, true, 0);
```

Comment:
This is supposed to stop the streaming? A comment or wrapper method would be useful here.

Reply:
Yeah, the last param is the volume, I will add a comment!